### DISCUSSION TITLE
Difference between release files?

### DISCUSSION THREAD

**Discussion by rashil2000** [rustdesk/rustdesk#1884] [General]
Sorry for ignoring the issue template, but I searched for a while in existing issues/discussions and couldn't find my answer. There are currently 3 files available to download (for my system) on the Releases page. 
1. https://github.com/rustdesk/rustdesk/releases/download/1.1.9/rustdesk-1.1.9-windows_x64-portable.zip
2. https://github.com/rustdesk/rustdesk/releases/download/1.1.9/rustdesk-1.1.9-windows_x64.zip
3. https://github.com/rustdesk/rustdesk/releases/download/1.1.9/rustdesk-1.1.9-x64.exe

I extracted the first two archives and found out that they contain the same exe (just with different name) - because their SHA256 hash is same. 

What's the difference between these three exes? Does changing the name change anything? Is the config file location same or different for all three?

Thanks in advance!

**Comment by rashil2000** [comment]
@rustdesk Okay so I figured out that appending a `-putes` to any of the exes makes it ask for installation, when run. That's the difference between (1) and (2). I couldn't figure out the difference between (2) and (3) though.

**Reply by bdntv** [reply]
Ones a zip archive, one is an executable.

**Reply by rashil2000** [reply]
Obviously 😅

I'm talking about the final extracted exe file

**Reply by bdntv** [reply]
Lol
